### PR TITLE
Blockchain Compute Docs Addition

### PR DESCRIPTION
**Pull request by olliegilbey**
This commit adds the documentation around using Blockchain Compute. Makes minor changes to the styling of docs.
Adds docs for both the UI and CLI approach to using BCC.

### CODE REVIEW

File: docs/blockchain-compute/usage/via-command-line/preparations.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+## Requirements
+
+To be a consumer, you only need access to the Cudos blockchain, an SSH key, and some CUDOS tokens.
+
+:::info
+If you simply want to purchase a VM, and would prefer to use a UI, you can use the [website](https://compute.cudos.org).
+
+This documentation is only needed if you want to create your own integration with the contract, or just want to learn how the system works.
+:::
+
```
Comment (jgtormo) [inline docs/blockchain-compute/usage/via-command-line/preparations.md:17]: should we also mention that the testnet contract can only be used if they are whitelited by a Cudos admin?

File: docs/blockchain-compute/usage/via-command-line/preparations.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+export RPC_NODE_TN="https://rpc.testnet.cudos.org:443"
+export CHAIN_ID_TN=cudos-testnet-public-3
+export GAS_TN=auto
+export GAS_PRICES_TN=5000000000000acudos
+export GAS_ADJUSTMENT_TN=1.3
+export KEYRING_TN=os
+export CONTRACT_ADDRESS_TN=cudos1nzu5uq00dny8fjs83eerjfv8l9qnntd9h03ag9hcl9tpf9upr0fshnafnk
+
+# environment variables for mainnet
+export RPC_NODE="https://rpc.cudos.org"
```
Comment (jgtormo) [inline docs/blockchain-compute/usage/via-command-line/preparations.md:46]:
```suggestion
export RPC_NODE="https://rpc.cudos.org:443"
```

File: docs/blockchain-compute/usage/via-command-line/preparations.md (hunk 1 above)
Comment (jgtormo) [reply docs/blockchain-compute/usage/via-command-line/preparations.md:17]: actually, let's remove the testnet as we'll likely never whitelist anyone external

**Review by ahmeddica — APPROVED** [review]
lgtm

**Review by ahmeddica — APPROVED** [review]

**Pull request merged**